experimental/generation/generator/templates/standard/standard.schema

```diff
@@ -95,7 +95,6 @@
         "PROPERTYName"
     ],
     "$templates": [
-        "booleanEntity",
         "library-HelpIntent.lu",
```

Comment:
How does this get brought in to support confirmation which is a part of every generated dialog?

Reply:
I think you can remove all the boolean $entities from above and this should be fine.  They get picked up because of the implicit $entities of boolean.


---
In reply to: [455941471](https://github.com/microsoft/BotBuilder-Samples/pull/2594#discussion_r455941471) [](ancestors = 455941471)